### PR TITLE
Use "local cache" instead of "local" when printing package graph

### PR DESCRIPTION
**Pull request by danimtb**
- [x] Refer to the issue that supports this Pull Request: #2333
- [x] If the issue has missing info, explain the purpose/use case/pain/need that covers this Pull Request.
- [x] I've read the [Contributing guide](https://raw.githubusercontent.com/conan-io/conan/develop/.github/CONTRIBUTING.md).
- [x] I've followed the PEP8 style guides for Python code.
- [ ] I've opened another PR in the Conan docs repo to the ``develop`` branch, documenting this one. Also adding a description of the changes in the ``changelog.rst`` file. https://github.com/conan-io/docs

- Changed "local" with "local cache" message.
- Added ' ' to refer to packages names or patterns in messages
- Added tests for Printer methods messages (as output from command line, not unit test of Printer class)
- Changed message "Package_ID:" to "Package ID:" and "outdated from..." to "Outdated from..."

Please **review the tests** as I may have done some unnecessary steps. Maybe the tests should be placed in commands directory. **Tests only covers Printer class messages** outputed in commands, not exception messages or others.

I think PR to Docs is not needed.

### CODE REVIEW

File: conans/client/printer.py [hunk 1]
```diff
@@ -197,7 +197,7 @@ def print_search_packages(self, packages_props, reference, recipe_hash, packages
         self._out.info("Existing packages for recipe %s:\n" % str(reference))
         # Each package
         for package_id, properties in sorted(packages_props.items()):
-            self._print_colored_line("Package_ID", package_id, 1)
+            self._print_colored_line("Package ID", package_id, 1)
```
Comment (memsharded) [inline conans/client/printer.py]: Though not pretty, some users might be relying on parsing the search output to get package IDs. I'd say, leave it as-is.

File: conans/test/functional/printer_test.py [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    requires = "bbb/0.1@lasote/channel"
+    exports = "txt_file.txt"
+
+"""
+
+txt_file = """
+    example file
+    """
+
+test_profile = """
```
Comment (memsharded) [inline conans/test/functional/printer_test.py]: Packages don't have settings. Profile irrelevant, don't use it

File: conans/test/functional/printer_test.py [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+        self.client.run("upload aaa/0.1@lasote/channel -r default")
+        self.client.run("remove -f aaa/0.1@lasote/channel")
+        self.client.run("export conanfile_libB.py lasote/channel")
+        self.client.run("upload bbb/0.1@lasote/channel -r default")
+        self.client.run("remove -f bbb/0.1@lasote/channel")
+        self.client.run("export conanfile_libC.py lasote/channel")
+
+    def print_graph_test(self):
+        self.client.run("install ccc/0.1@lasote/channel",
+                        ignore_error=True) #Ignore ERROR: Missing prebuilt package
```
Comment (memsharded) [inline conans/test/functional/printer_test.py]: If ignore_error, capture client.run() return and check it is indeed an error. Also check output Error message why this is an error. It is weird, try to do the test without error.

File: conans/test/functional/printer_test.py [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+        self.assertIn("aaa/0.1@lasote/channel from 'default'", self.client.out)
+        self.assertIn("bbb/0.1@lasote/channel from 'default'", self.client.out)
+        self.assertIn("ccc/0.1@lasote/channel from local cache", self.client.out)
+
+    def print_info_test(self):
```
Comment (memsharded) [inline conans/test/functional/printer_test.py]: What are you testing here? Not clear, I'd say this test is not necessary.

File: conans/test/functional/printer_test.py [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+    Remote: None"""
+        ccc_require_output = """Required by:
+        None
+    Requires:
+        bbb/0.1@lasote/channel"""
+        self.assertIn(bbb_require_output, self.client.out)
+        self.assertIn(ccc_header_output, self.client.out)
+        self.assertIn(ccc_require_output, self.client.out)
+
+    def print_search_recipes_test(self):
```
Comment (memsharded) [inline conans/test/functional/printer_test.py]: Too much ``setUp()`` for this simple test. I also think that this functionality should be already in existing tests.

File: conans/client/printer.py (hunk 1 above)
Comment (danimtb) [reply conans/client/printer.py]: Don't like "Package_ID" as it is not a parameter nor a variable, but I understand the reason

File: conans/test/functional/printer_test.py (hunk 2 above)
Comment (danimtb) [reply conans/test/functional/printer_test.py]: The profile is used to test `Printer.print_profile()` output

File: conans/test/functional/printer_test.py (hunk 4 above)
Comment (danimtb) [reply conans/test/functional/printer_test.py]: Testing alll `Printer.print_info()` output. Specially for "Requires:" and "Required by:" fields depending on the reference used

File: conans/test/functional/printer_test.py (hunk 5 above)
Comment (danimtb) [reply conans/test/functional/printer_test.py]: These tests only check output messages for printer, some messages are checked in other tests but not all of them. I could reduce the asserts done to keep this smaller

**Comment by danimtb** [comment]
@memsharded I made complete tests for all methods of the Printer class but I could test only the fixed message if that is the desired approach for tests

File: conans/test/functional/printer_test.py (hunk 2 above)
Comment (memsharded) [reply conans/test/functional/printer_test.py]: This is part of the confusion that this tests is testing other unrelated things to this fix.
Refactoring and improving tests is something to do, but I'd say better in a different PR, referencing the other tests that should cover it and are not doing it.

**Pull request merged**